azure-templates/deploy-steps.yml

```diff
@@ -26,7 +26,33 @@ parameters:
   - name: 'appName'
     type: string
     default: ''
-    
+
+  # full endpoint to perform healthcheck against, 
+  - name: 'healthcheckEndpoint'
+    type: string
+
+  # min container instance cpu
+  - name: 'containerInstanceCpu'
+    type: string
+    default: '0.5'
+
+  # min container instance memory
+  - name: 'containerInstanceMemory'
+    type: string
+    default: '0.5'
```

Comment:
default value for `rest-healthcheck/template.yaml` we can remove it?

Reply:
ok